app/src/main/java/org/wikipedia/feed/model/UtcDate.kt

```diff
@@ -0,0 +1,42 @@
+package org.wikipedia.feed.model
+
+import java.util.Calendar
+import java.util.TimeZone
+
+class UtcDate(age: Int) {
+    private val cal: Calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"))
+    private val year: String
+    private val month: String
+    private val date: String
+
+    fun baseCalendar(): Calendar {
+        return cal
+    }
+
+    fun year(): String {
+        return year
+    }
+
+    fun month(): String {
+        return month
+    }
+
+    fun date(): String {
+        return date
+    }
+
+    init {
+        cal.add(Calendar.DATE, -age)
+        year = cal[Calendar.YEAR].toString()
+        month = pad((cal[Calendar.MONTH] + 1).toString())
+        date = pad(cal[Calendar.DATE].toString())
+    }
+
+    companion object {
+        private fun pad(value: String): String {
```

Comment:
This function is no longer necessary. Kotlin has this function built in.

Reply:
Thanks for suggestion. IDK Kotlin has built-in `padStart` method. 
Updated via 4ae3b1e